tilelang/contrib/nvcc.py

```
@@ -405,21 +405,17 @@ def get_target_compute_version(target=None):
-                major = arch[0:2]
-                minor = arch[2]
-                return major + "." + minor
-            else:
-                # This is for arch like "sm_90a"
-                major, minor, suffix = arch
-            return major + "." + minor + "." + suffix
+            major = arch[0:2]
+            minor = arch[2]
+            return major + "." + minor
+        else:
+            raise ValueError("Unsupported arch: {}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)